deploy/dependencies.sh

```diff
@@ -1,19 +1,5 @@
 #!/bin/bash
 
```

Comment:
You can add descriptive header for this shell script. Sample script header can be seen [here](https://github.com/AutolabJS/AutolabJS/blob/master/tests/test_runner.sh)